### PR TITLE
Better documentation of configuration options

### PR DESCRIPTION
**Pull request by bovine3dom**
It looks like TypeDoc has a bug where object keys such as `<C-a>` get mangled into HTML tags, which we will need to fix before merging this.

I've also made the main help page a little worse, I think. It would be nice to fix that too.

### CODE REVIEW

**Comment by bovine3dom** [comment]
I finally got around to filing an issue: https://github.com/TypeStrong/typedoc/issues/780

**Comment by glacambre** [comment]
I've opened a PR in order to unblock this: https://github.com/TypeStrong/typedoc-default-themes/pull/58

**Comment by bovine3dom** [comment]
@glacambre can we just edit `package.json` to point to your [repo](https://github.com/glacambre/typedoc-default-themes/tree/fix_wierd_member_names)?

**Comment by glacambre** [comment]
Sure, I think you need something like this:
```
    "typedoc": {
      "version": "^0.9.0",
      "dependencies": {
        "typedoc-default-themes": "git://github.com/glacambre/typedoc-default-themes#fix_wierd_member_names" /* Yes, with the typo */
      }
    },
```

I couldn't test it on your PR though, I couldn't get TypeDoc to generate documentation for objects.

**Comment by bovine3dom** [comment]
> I couldn't test it on your PR though, I couldn't get TypeDoc to generate documentation for objects.

:S TypeDoc generates the documentation fine for me. You just want to visit `_config` rather than `_excmds` in the help page.

I can't figure out how to tell NPM to use your package, though. I've tried various combinations of putting your invocation in `package{,_lock}` and using `github:glacambre/...` as we do for web-ext-types but I've run out of time for Tridactyl today.

File: src/static/typedoc/assets/css/main.css [hunk 1]
```diff
@@ -2993,6 +3002,7 @@ ul.TridactylSetting {
     font-weight: bold;
     font-size: 10pt;
 }
+>>>>>>> e1bb76600e8ebc796fe61818217a7ddab59770f9
```
Comment (bovine3dom) [inline src/static/typedoc/assets/css/main.css]: ...no wonder the CSS looked a bit dodgy

File: src/static/typedoc/assets/css/main.css [hunk 2]
```diff
@@ -3003,10 +3013,46 @@ ul.TridactylSetting li {
     border-radius: 3px;
 }
 
+<<<<<<< HEAD
```
Comment (bovine3dom) [inline src/static/typedoc/assets/css/main.css]: blarg

**Comment by glacambre** [comment]
The problem was that I had made the (wrong) assumption that npm would build the packages it downloaded. It turns out that it doesn't. I pushed a built version to the "fix_weird_member_names_bin" branch of my repo and just adding
```
"typedoc-default-themes": "git://github.com/glacambre/typedoc-default-themes#fix_weird_member_names_bin"
```
to package.json seems to be enough to use my version, which indeed fixes the issue.

**Pull request merged**